### ISSUE TITLE
Togglable tabs within dropdown closes the dropdown when changing the tabs

### ISSUE THREAD

**Issue by belchugov** [twbs/bootstrap#17783]
Hi, I'm trying to set toggable tabs under a dropdown menu navigation. When changing the tabs, the dropdown closes, the tab is changed but the focus is lost.
Example here: http://www.bootply.com/x0d0L4gk8a

**Comment by twbs-lmvtfy** [comment]
Hi @belchugov!

You appear to have posted a live example (http://s.bootply.com/render/x0d0L4gk8a), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 60, column 13 thru column 17: Element ```` div ```` not allowed as child of element ```` ul ```` in this context. (Suppressing further errors from this subtree.)

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by belchugov** [comment]
Fix the small bug:
http://s.bootply.com/render/x0d0L4gk8a

**Comment by twbs-lmvtfy** [comment]
Hi @belchugov!

You appear to have posted a live example (http://s.bootply.com/render/x0d0L4gk8a), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 60, column 13 thru column 17: Element ```` div ```` not allowed as child of element ```` ul ```` in this context. (Suppressing further errors from this subtree.)

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by belchugov** [comment]
Fix small bug:
http://www.bootply.com/uM2ic0QcZC

**Comment by mdo** [comment]
[**Bootstrap 3 is no longer being officially developed or supported.**](https://github.com/twbs/bootstrap/issues/20631)

All work has moved onto our next major release, v4. As such, this issue or pull request is being closed as a "won't fix." For additional help and support, we recommend [utilizing our community resources](https://github.com/twbs/bootstrap/blob/master/README.md#community). Thanks for your understanding, and see you on the other side of v4!

<3,
@mdo and team
